content-api-post.js

```diff
@@ -56,6 +60,9 @@ exports.handler = function (event, context, callback) {
         SalesforceAccountId: data.accountId,
       };
       // All remaining metadata keys are optional. We can't append `null`s as this is not a valid value for headers.
+      if (data.championFundId) {
```

Comment:
@tbg-jasonfung  what about the path? This change on its own surely doesn't work - the most critical part (where the file lives) still relies on account ID.

Reply:
@NoelLH Thanks - I've kept the accountId and championFundId metadata dynamic, I think it'll be easier to have the Ids set the correct metadata instead of relying on account Id